etcd/etcd-env.sh

```diff
@@ -6,11 +6,11 @@ if test x$NAME == x; then
 fi
 export ETCD_NAME=${ETCD_NAME:-$HOSTNAME}
 export ETCD_DATA_DIR=/var/lib/etcd/${NAME}.etcd
```

Comment:
OK, now it uses the data dir value that's in etcd.conf, which is set in the package to be `/var/lib/etcd/default.etcd`. You can change that to another value, as long as it's under `/var/lib/etcd`. I tried to set the default in `manifest.json` to be based on $NAME, but either it wasn't getting populated correctly, or it was getting overwritten by the default in the config file.